### ISSUE TITLE
Build our own PRG / disk-image

### ISSUE THREAD

**Issue by Kroc** [Kroc/EliteDX#19]
Right now, we're dependent on the GMA86 loader with it's copy-protection and encryption. Before we can really get into changing the game we need to be able to construct our own single-file program, stripping out the GMA86 fast-loader / relocation / encryption code. This requires a massive re-ordering of the program's layout and will likely break a lot of code that hasn't been properly disassembled yet.

This is a separate issue from adding out own fast-loader, that can be done once we can build our own PRG files.

- [ ] Produce a new bootsrap / once-off initialisation code -- the loader currently handles calling a handful of routines to initialise the game and jump into the entry point. Once-off init code can live in $0400-$0700 as this area is wiped and used as scratch RAM by the game
